src/views/ActivityAppNavigation.vue

```diff
@@ -49,14 +49,11 @@ import { translate as t } from '@nextcloud/l10n'
 import { generateUrl } from '@nextcloud/router'
 import { computed, ref } from 'vue'
 
-import {
-	NcAppNavigation,
-	NcAppNavigationItem,
-	NcAppNavigationSettings,
-	NcButton,
-	NcCheckboxRadioSwitch,
-} from '@nextcloud/vue'
-
+import NcAppNavigation from '@nextcloud/vue/dist/Components/NcAppNavigation.js'
+import NcAppNavigationItem from '@nextcloud/vue/dist/Components/NcAppNavigationItem.js'
```

Comment:
Any reason?

Reply:
On master the js/ size is 8.6M, with this small adjustment we only import what we need, bringing the package size down to 6.0M in js/